test/utils/ui.test.helper.ts

```diff
@@ -7,3 +7,63 @@ export const clearInput = async (page: puppeteer.Page, selector: string) => {
   }, selector);
   await page.waitForSelector(selector);
 };
+
+export const reloadPage = async (page: puppeteer.Page, pageUrl: string) => {
+  let url = page.url();
+  if (url !== pageUrl) {
+    throw new Error('Url is not correct!');
+  }
+  await page.reload({ waitUntil: ['networkidle0', 'domcontentloaded'] });
+};
+
+export const verifyUserIsOnPageByTextElements = async (
+  page: puppeteer.Page,
+  selector: string,
+  textElement: string
+) => {
+  await page.waitForSelector(selector);
+
+  await page.waitForSelector(selector);
+  const returnedText = await page.$$eval(selector, element => {
+    return element.map(element => element.textContent?.trim()).join(' ');
+  });
+
+  return returnedText === textElement;
+};
+
+export const verifyUserIsOnPageByJoinTextElements = async (
+  page: puppeteer.Page,
+  selector: string
+) => {
+  await page.waitForSelector(selector);
+
+  await page.waitForSelector(selector);
+  const returnedText = await page.$$eval(selector, element => {
+    return element.map(element => element.textContent?.trim()).join(' ');
+  });
+
+  return returnedText;
+};
+
+export const verifyUserIsOnPageByGetTextElement = async (
```

Comment:
if you think 3 functions that do 90% the same are needed, at least wrap the similar parts in a function